pkg/sdkserver/sdkserver_test.go

```diff
@@ -1377,7 +1367,7 @@ func TestSDKServerGracefulTerminationInterrupt(t *testing.T) {
 	wg.Add(1)
 
 	go func() {
-		err = sc.Run(sdkCtx)
+		err := sc.Run(sdkCtx)
 		assert.Nil(t, err)
 		wg.Done()
 	}()
```

Comment:
These is a comment way down on line 1560 that needs to be adjusted as well, since it refers to the feature gate.